Controller/GravatarController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Ornicar\GravatarBundle\Controller;
+
+
+use Ornicar\GravatarBundle\Exception\ImageTransferException;
+use Ornicar\GravatarBundle\GravatarApi;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class GravatarController extends Controller
+{
+    public function getImageAction($hash, $size, $rating, $default, $secure)
```

Comment:
normally in symfony i think we just use `imageAction` or in this case the best name is proberly `showAction`